sql/core/src/main/scala/org/apache/spark/sql/functions.scala

```diff
@@ -8368,8 +8368,13 @@ object functions {
    * @since 3.5.0
    */
   @scala.annotation.varargs
-  def call_function(funcName: String, cols: Column*): Column =
-    withExpr { UnresolvedFunction(funcName, cols.map(_.expr), false) }
+  def call_function(funcName: String, cols: Column*): Column = withExpr {
```

Comment:
shall we add a private method that takes a `Seq[String]`, so that we can call it if a method does not want to support qualified names?